pkg/cmd/cli/describe/describer.go

```diff
@@ -214,7 +214,12 @@ func describeBuildSpec(p buildapi.BuildSpec, out *tabwriter.Writer) {
 			formatString(out, "Binary", "provided on build")
 		}
 	}
-
+	for _, image := range p.Source.Images {
+		formatString(out, "Files from", fmt.Sprintf("%s/%s", image.From.Namespace, image.From.Name))
```

Comment:
that function wouldn't actually help here.  it would just go through the "neither has a slash, concatenate with a slash between them" path.

will create a new helper.